app/pages/Page.js

```diff
@@ -15,17 +15,19 @@ type Props = TranslatorProps & PassedProps;
 
 const Page = (props: Props): React.Node => {
   return (
-    <div>
+    <React.Fragment>
       <NavigationBar />
-      <Grid stretched={true} relaxed="very">
-        <Grid.Column width={1} />
-        <Grid.Column stretched={true} width={14}>
-          <Container>
-            {props.children}
-          </Container>
-        </Grid.Column>
-      </Grid>
-    </div>
+      <div className="page-layout__grid">
+        <Grid stretched={true}>
+          <Grid.Column width={1} />
```

Comment:
Page is still not completely centered at small viewport sizes, probably because of this unnecessary column?

Reply:
Looks centered to me. I put it there initially for some padding on the left, but I can also do that with LESS now.

Reply:
One sec, let me take a screenshot :)

Reply:
![not-centered](https://user-images.githubusercontent.com/9011748/38312444-0fdf235e-3822-11e8-9cfc-d9b3037b429b.jpg)

^ Happens at the smallest fluid size, right before the page becomes fixed-width.